src/test/java/edu/ucsb/cs156/happiercows/controllers/CommonsControllerTests.java

```diff
@@ -1235,6 +1238,28 @@ public void UpdateCommonsTest_withBoundaryParameters() throws Exception {
         assertEquals(100, commonsPlus.getEffectiveCapacity());
     }
 
+    @WithMockUser(roles = {"USER"})
```

Comment:
is there any difference for admin?

Reply:
I think this should be ok.  When tbings are working properly, every ADMIN also has the role USER.